content/sensu-enterprise/3.1/installation/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Installation Overview"
+description: "Sensu Enterprise installation resources"
+weight: 1
+product: "Sensu Enterprise"
+version: "3.1"
+platformContent: true
+platforms: ["RHEL/CentOS", "Ubuntu/Debian"]
+menu:
+  sensu-enterprise-3.1:
+    parent: installation
+---
+
+- [Installing Sensu Enterprise](#installing-sensu-enterprise)
+- [Upgrading from Sensu Core to Sensu Enterprise](#upgrading-from-sensu-core-to-sensu-enterprise)
+
+## Installing Sensu Enterprise
+
+Sensu Enterprise is supported on RHEL/CentOS and Ubuntu/Debian Linux
+distributions. Please see the relevant section below for information
+on installing Sensu Enterprise and its prerequisites.
+
+Prior to working through either the installation or upgrade
+instructions linked below, you'll need to obtain your Sensu Enterprise
+repository credentials via the [Sensu Account Manager
+portal][1]. These credentials are comprised of
+a username and password which are required to access Sensu Enterprise
+software packages.
+
+_**NOTE: manual installation is recommended for pre-production environments
+only.** Please note that this guide is not intended to provide instructions for
+deploying Sensu Enterprise into "production" environments. Production
+deployment strategies &ndash; including using automation tools like
+Chef, Puppet, or Ansible to install and configure Sensu &ndash; are
+be discussed [at the conclusion of the Sensu Core installation guide][2]._
+
+{{< platformBlock "RHEL/CentOS" >}}
+### Installing Sensu Enterprise and prerequisites on RHEL/Centos
+
+Sensu Enterprise builds on the same [architecture][arch] as Sensu Core.
+
+If you're a new Sensu user getting started with Sensu Enterprise,
+please complete the following selected sections of the [Sensu installation guide][3]:
+
+1. [Install and Configure Redis](/sensu-core/latest/installation/install-redis-on-rhel-centos)
+2. [Install and Configure RabbitMQ](/sensu-core/latest/installation/install-rabbitmq-on-rhel-centos)
+3. [Install Sensu Enterprise](/sensu-core/latest/platforms/sensu-on-rhel-centos/#sensu-enterprise)
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Ubuntu/Debian" >}}
+### Installing Sensu Enterprise and prerequisites on Ubuntu/Debian
+
+Sensu Enterprise builds on the same [architecture][arch] as Sensu Core.
+
+If you're a new Sensu user getting started with Sensu Enterprise,
+please complete the following selected sections of the [Sensu installation guide][3]:
```

Comment:
I'd suggest removing this link (for both platforms) since we really want users to use the links listed below instead. If we do want to keep this link, we should change it to link to installation/overview instead of installation/summary.